### ISSUE TITLE
Sarkhan the Masterless and Mox Amber Interaction 

### ISSUE THREAD

**Issue by scmurray15** [magefree/mage#5889]
When you plus Sarkhan to make the the Planeswalkers into Dragons, they currently lose their legendary status, and Mox therefore doesnt work.

**Comment by Zerrisx** [comment]
Yep; Mox Amber will only tap for red (Sarkhan's color), regardless of any other planeswalker-creatures in play.  Although they are still being listed as Legendary for me, so the issue may be deeper.

![image](https://user-images.githubusercontent.com/59861365/87571797-2ebdc380-c690-11ea-8857-b7097a5fa262.png)
